### ISSUE TITLE
How can I create my own pretrained model?

### ISSUE THREAD

**Issue by SarahTeddyToasty** [junyanz/pytorch-CycleGAN-and-pix2pix#1094]
I'm looking to turn sunny images into rainy images, and I assume that using one of the pretrained models would not have a good result. How can I create my own? I found your [site with the pretrained models](http://efrosgans.eecs.berkeley.edu/cyclegan/pretrained_models/) but I don't know how to see the code using Google Colab.

**Comment by junyanz** [comment]
You need to train your model following these [steps](https://github.com/junyanz/pytorch-CycleGAN-and-pix2pix#cyclegan-traintest).

**Comment by SarahTeddyToasty** [comment]
What do I put into the --name flag?

**Comment by junyanz** [comment]
See the [options](https://github.com/junyanz/pytorch-CycleGAN-and-pix2pix/blob/master/options/base_options.py#L24).

**Comment by SarahTeddyToasty** [comment]
Oh so that's just where it gets stored? So where exactly does your pretrained models get called?

**Comment by junyanz** [comment]
See the [tutorial](https://github.com/junyanz/pytorch-CycleGAN-and-pix2pix#apply-a-pre-trained-model-cyclegan) on how to apply a pre-trained model. It will load the model from the directory specified by `--checkpoints_dir` and `--name`.

**Comment by SarahTeddyToasty** [comment]
So to train my own model I need to run the train.py, and if you're using your pre-trained model you can bypass the train.py and just download the script and run test.py?

**Comment by junyanz** [comment]
Yes. you first need to train a model using `train.py`, and then test it using `test.py`. You don't need to use the downloading script. (The script downloads a model in our model zoo). See the code [overview](https://github.com/junyanz/pytorch-CycleGAN-and-pix2pix/blob/master/docs/overview.md) for more details.

**Comment by SarahTeddyToasty** [comment]
Thank you. Also, I'm using Google Colab and I'm trying to install wisdom. But it's been loading for a long time, so I'm not sure how to proceed. 
![image](https://user-images.githubusercontent.com/66890934/87705713-52eed280-c76c-11ea-8163-08845d226156.png)
